### ISSUE TITLE
ImageFIeld validation

### ISSUE THREAD

**Issue by sazarkin** [encode/django-rest-framework#5534]
## Checklist

- [x] I have verified that that issue exists against the `master` branch of Django REST framework.
- [x] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [x] This is not a usage question. (Those should be directed to the [discussion group](https://groups.google.com/forum/#!forum/django-rest-framework) instead.)
- [x] This cannot be dealt with as a third party library. (We prefer new functionality to be [in the form of third party libraries](http://www.django-rest-framework.org/topics/third-party-resources/#about-third-party-packages) where possible.)
- [x] I have reduced the issue to the simplest possible case.
- [x] I have included a failing test as a pull request. (If you are unable to do so we can still accept the issue.)

## Descriptions
In this release[https://docs.djangoproject.com/en/1.11/releases/1.11/#models] Django added default validator for ImageField.
This validation steps is ignored in DRF 

## Steps to reproduce
* Take png image and change extension .txt
* Upload

## Expected behavior

 ```ValidationError: [u"File extension 'txt' is not allowed. Allowed extensions are: 'ppm, grib, im, rgba, rgb, jpx, h5, jpe, jpf, jpg, fli, jpc, sgi, gbr, pcx, mpeg, jpeg, ps, flc, tif, hdf, icns, gif, palm, mpg, fits, pgm, mic, pxr, fit, xbm, eps, emf, jp2, dcx, webp, bmp, bw, pbm, j2c, psd, pcd, ras, j2k, mpo, cur, fpx, ftu, png, msp, iim, wmf, jfif, tga, bufr, ico, ftc, xpm, pdf, dds, tiff'."]```

## Actual behavior

It's valid.

## Suggestion

Add to ImageFIeld something like this:
```python
if hasattr(self._DjangoImageField, 'default_validators'):
            self.default_validators.extend(self._DjangoImageField.default_validators)
```

**Comment by carltongibson** [comment]
At first glance this seems reasonable. Fancy putting a PR together for this?

**Comment by sazarkin** [comment]
I've added pr

**Comment by carltongibson** [comment]
Closed by #5539
